soc/arm64/rockchip/rk3568/soc.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2022 HNU-ESNL
+ * Copyright 2022 openEuler SIG-Zephyr
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+
+#ifdef CONFIG_SOC_RK3568
+
+#ifndef CONFIG_PM_CPU_OPS_PSCI
+int pm_cpu_on(unsigned long cpuid, uintptr_t entry_point)
```

Comment:
why is this needed? If `CONFIG_PM_CPU_OPS_PSCI` is not defined, `pm_cpu_on` should return `-ENOTSUP` (if `CONFIG_PM_CPU_OPS` is defined).

Reply:
Thanks, it seem not necessary for specific scenarios. I think there may be situations where `CONFIG_PM_CPU_OPS_PSCI` is not defined but `pm_cpu_on` is used.